src/FeedbackBot/Controllers/HomeController.cs

```diff
@@ -50,21 +63,32 @@ public class HomeController : Controller
         public async Task<ActionResult> CreateIssue(string title, string description)
         {
             // Github Authentication
-            var client = new GitHubClient(new ProductHeaderValue("FeedbackBot"));
-            var basicAuth = new Credentials("UCDFeedbackBot", "99LinesOfCode"); 
-            client.Credentials = basicAuth;
-            var kerberos = User.Identity.Name;
+            var client = initialize();
 
             // Initialize new issue
             var createIssue = new NewIssue(title)
             {
-                Body = description + "\r\n--------------------\r\nVotes: 1\r\nVoters: " + kerberos,
+                Body = description + "\r\n--------------------\r\nVotes: 1\r\nVoters: " + Globals.kerberos,
```

Comment:
@cszhu two optional alternatives to concatenating strings (which is fine here if you want to leave it because it is simple).  use `string.format()` like `string.Format("Hello!  My name is {0} {1} and I am {2} years old.",
    person.FirstName, person.LastName, person.Age);` or the new fun way, use string interpolation `$"Hello!  My name is {person.FirstName} {person.LastName} and I am
    {person.Age} years old.";`